test/testdata/java_archived_book_name.baseline

```diff
@@ -101,6 +104,22 @@ public class ArchivedBookName extends BookName {
     return PATH_TEMPLATE.matches(formattedString);
   }
 
+  public Map<String, String> getFieldValuesMap() {
+    if (fieldMap != null) {
+      return fieldMap;
+    }
+    ImmutableMap.Builder<String, String> fieldMapBuilder = ImmutableMap.builder();
+    fieldMapBuilder.put("archiveId", archiveId);
+    fieldMapBuilder.put("bookId", bookId);
+
+    fieldMap = fieldMapBuilder.build();
+    return fieldMap;
+  }
+
+  public String getFieldValue(String fieldName) {
+    return fieldMap.get(fieldName);
```

Comment:
This could cause a NullPointerException - it needs to call getFieldValuesMap()

Reply:
Oh yes ok!